docs/advanced/other.md

```diff
@@ -0,0 +1,141 @@
+{!docs/setup-fastlane-header.md!}
+
+# Other
+
+Other advanced topics that didn't fit anywhere else:
+
+# Environment Variables
+
+You can define environment variables in a `.env` or `.env.default` file in the same directory as your `Fastfile`. Environment variables are loaded using [dotenv](https://github.com/bkeepers/dotenv). Here's an example:
+
+```no-highlight
+WORKSPACE=YourApp.xcworkspace
+HOCKEYAPP_API_TOKEN=your-hockey-api-token
+```
+
+_fastlane_ also has a `--env` option that allows loading of environment specific `dotenv` files. `.env` and `.env.default` will be loaded before environment specific `dotenv` files are loaded. The naming convention for environment specific `dotenv` files is `.env.<environment>`
+
+For example, `fastlane <lane-name> --env development` will load `.env`, `.env.default`, and `.env.development`
+
+Alternatively, as environment variables are not a fastlane specific thing, you can also use standard methods to set them:
+
+```
+DELIVER_USER="felix@krausefx.com" fastlane test
+```
+
+or
+
+```
+export DELIVER_USER="felix@krausefx.com";
+fastlane test
+```
+
+Although it kind of defeats the purpose of using them in the first place (not to have their content in any files), you can also set them in your `Fastfile`:
+
+```
+ENV["DELIVER_USER"] = "felix@krausefx.com"
+```
+
+# Adding Credentials
+
+You can add credentials for use by _fastlane_ to your keychain using the [CredentialsManager](https://github.com/fastlane/fastlane/tree/master/credentials_manager) command line interface. This is useful for situations like CI environments.
+
+**Adding a Credential**
+```no-highlight
+fastlane fastlane-credentials add --username felix@krausefx.com
+Password: *********
+Credential felix@krausefx.com:********* added to keychain.
+```
+
+**Removing a Credential**
+```no-highlight
+fastlane fastlane-credentials remove --username felix@krausefx.com
+password has been deleted.
+```
+
+# Manually manage the fastlane match Repo
```

Comment:
This should probably be part of the `match.md` README in the `fastlane/fastlane` repo instead, so it's generated as part of `actions/match`

Reply:
Good point, let's create an issue for it though and get this merged here so the changes are not mixed?